mypyc/test-data/run-misc.test

```diff
@@ -875,6 +875,89 @@ assert call_all(mixed_110) == 1
 assert call_any_nested([[1, 1, 1], [1, 1], []]) == 1
 assert call_any_nested([[1, 1, 1], [0, 1], []]) == 0
 
+
+[case testSum]
+[file driver.py]
+print('test sums of numbers')
```

Comment:
These don't actually test anything, since the `driver.py` file is not compiled. (This is a common mistake so we should try to improve the developer experience here.) 

The preferred way is to use `test_` functions in the main test case and not include `driver.py` at all. For example:

```py
[case testFoo]
def test_whatever() -> None:
    assert 1 + 2 == 3

def test_more() -> None:
    assert 'x' * 2 == 'xx'
```

Reply:
Thanks for the tip!